var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,32 +80,57 @@ def install(self, spec, prefix):
             optflags = {
                 'gcc': ['-O2',
                         '-ffast-math',
-                        '-ffree-form',
-                        '-ffree-line-length-none',
                         '-ftree-vectorize',
                         '-funroll-loops',
                         '-mtune=native'],
                 'intel': ['-O2',
                           '-pc64',
                           '-unroll',
-                          '-heap-arrays 64']
+                          '-xHost']
```

Comment:
Well spotted. So I think I am in a dilemma: this is in `cp2k` Makefiles, and seems (according to our users) to speed-up things. But you are right that it may prevent cross-compilation in the future. A variant doesn't seem a good idea, nor removing it and passing the word that it is better to install with:
```
spack install cp2k%intel cflags="-xHost" cxxflags="-xHost" fcflags="-xHost"
```
Should I try with a check that the current platform is what we are installing for? Or do you have other ideas?